app/models/organization.rb

```diff
@@ -181,6 +188,20 @@ def bike_shop_display_integration_alert?
     bike_shop? && %w[no_pos broken_pos].include?(pos_kind)
   end
 
+  def bikes_in_region_counts
+    return unless regional?
+    return @bikes_in_region_counts if defined?(@bikes_in_region_counts)
+
+    bikes_in_orgs = regional_suborganizations.includes(:bikes).flat_map(&:bikes).map(&:id)
```

Comment:
I would like `bikes_in_orgs` and `bikes_in_region` as separately queryable active record collections, because the eventual goal here is to add graphs, time series, etc, which will use scopes to display counts for specific conditions.